omg_ros_nav_bridge/RobotPose goal
omg_ros_nav_bridge/RobotPose state
omg_ros_nav_bridge/Obstacle[] dyn_obstacles
float32 current_time
